connector-packager/connector_packager/connector_file.py

```diff
@@ -12,8 +12,8 @@ def extension(self) -> str:
         # Remove the leading period.
         return ext if not ext else ext[1:]
 
-    def __lt__(self, other):
+    def __lt__(self, other: 'ConnectorFile') -> bool:
```

Comment:
this is a [forward reference](https://www.python.org/dev/peps/pep-0484/#forward-references)